generators/app/index.js

```diff
@@ -276,8 +280,11 @@ module.exports = yeoman.Base.extend({
     // fix the two files that use $http instead of resource
       this.template('jhipster/_auth.jwt.service.js', 'app/main/jhipster/services/auth/auth.jwt.service.js');
       this.template('jhipster/_profile.service.js', 'app/main/jhipster/services/profiles/profile.service.js');
-
-
+      //social login fix
```

Comment:
Use `this.enableSocialSignIn` around the social files and `this.enableWebsocket` around the tracker service